examples/raster_management.py

```diff
@@ -21,8 +21,18 @@
 # Get the first page applying a filter
 for raster in client.list_rasters(folder_id):
     pprint("raster %s" % "\n".join(["%s=%s" % item for item in raster.items()]))
-# Upload and removal
+# Upload, edition and removal
 local_raster_id = client.upload_raster("data/raster1.tif", name="A short-lived raster")
 print("Uploaded a second local raster=", local_raster_id)
+# Editing the image's band specification. See https://docs.picterra.ch/imagery/#Multispectral
+client.edit_raster(local_raster_id, multispectral_band_specification={
+    "ranges": [
+        [0, 128], [0, 128], [0, 128]
+    ],
+    "display_bands": [
+        {"name": "default", "bands": [2, 1, 0]}
```

Comment:
nit: would be worth to add the `type` key?

Reply:
Good catch! Added